LonaStudio/Workspace/InspectorView.swift

```diff
@@ -501,6 +508,22 @@ class InspectorView: NSStackView {
         return backgroundSection
     }
     
+    func renderShadowSection() -> DisclosureContentRow {
+        shadowEnabledView.imagePosition = .imageOnly
+        
+        let shadowContainer = NSStackView(
```

Comment:
Since `View`s don't support shadows yet, we should hide this like we do for a few of the other properties: https://github.com/airbnb/Lona/blob/master/LonaStudio/Workspace/InspectorView.swift#L665